_posts/2021-07-07-cockpit-248.md

```diff
@@ -0,0 +1,68 @@
+---
+title: Cockpit 248
+author: garrett
+date: '2021-07-07'
+tags: cockpit, machines, ostree
+slug: cockpit-248
+category: release
+summary: ''
+---
+
+Cockpit is the [modern Linux admin interface](https://cockpit-project.org/).
+We release regularly.
+
+Here are the release notes from Cockpit 248, Cockpit-Machines 247, and Cockpit-OSTree 187:
+
+
+## Metrics: Install missing packages
+
+[Cockpit 247](https://cockpit-project.org/blog/cockpit-247.html) introduced setting up a machine for Grafana metrics. However, this only worked if the `pcp` and `redis` packages were already installed. With this Cockpit version, the options will always work, and missing packages get installed as needed.
```

Comment:
unclear which "this Cockpit version" is, following as it is, directly after mentioning another cockpit version.

Reply:
I'm happy to change the original PR to "248". Does that still make sense, or is this PR now the primary thing?

Reply:
Updated in https://github.com/cockpit-project/cockpit/pull/16009 , with s/this Cockpit version,/Cockpit version 248/ (mind the dropping of the comma)

Reply:
I already edited a lot; I don't want to remerge, rebase, fix conflicts yet again. Addressed in a commit.